artifactory/services/discardBuilds.go

```diff
@@ -37,10 +37,8 @@ func (ds *DiscardBuildsService) DiscardBuilds(params DiscardBuildsParams) error
 		return err
 	}
 	requestFullUrl += "?async=" + strconv.FormatBool(params.IsAsync())
-
-	buildRepo := utils.BuildRepoNameFromProjectKey(params.ProjectKey)
-	if buildRepo != "" {
-		requestFullUrl += "&buildRepo=" + buildRepo
+	if params.ProjectKey != "" {
+		requestFullUrl += "&project=" + params.ProjectKey
```

Comment:
```suggestion
	requestFullUrl += utils.GetProjectQueryParam(params.ProjectKey)
```

Reply:
@yahavi,
The ```utils.GetProjectQueryParam(params.ProjectKey)``` will not worl here, because we need the query param to start with an ampersand. The the below function code - 
```
func GetProjectQueryParam(projectKey string) string {
	if projectKey == "" {
		return ""
	}
	return "?project=" + projectKey
}
```